windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -150,10 +150,8 @@ $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";"
 # unneeded docs that bloat the file size (and also seem to cause issues with unzipping).
 ###############################
 
-# Freeze serverengine version, as there is a compatibility issue with later versions.
-# See: https://github.com/fluent/fluentd/issues/1195.
-& $GEM_CMD install serverengine:1.6.4 fluentd:0.14.1 --no-ri --no-rdoc --no-document
-& $GEM_CMD install windows-pr:1.2.5 win32-ipc:0.7.0 win32-event:0.6.3 win32-eventlog:0.6.6 win32-service:0.8.9 fluent-plugin-winevtlog:0.0.4 --no-ri --no-rdoc --no-document
+& $GEM_CMD install fluentd:0.14.14 --no-ri --no-rdoc --no-document
+& $GEM_CMD install windows-pr:1.2.5 win32-ipc:0.6.6 win32-event:0.6.3 win32-eventlog:0.6.6 win32-service:0.8.9 fluent-plugin-winevtlog:0.0.4 --no-ri --no-rdoc --no-document
```

Comment:
I just pinned it to the newest the first time, but somewhere in the dependency line for fluentd 0.14.14 it required 0.6.6.